hw6/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu
+
+CMD ["echo","$PWD"]
```

Comment:
I am not sure why you are using this line of code. Seems uneccessary

Reply:
I was trying to see if I can get the current working directory path using this.